lib/ipmi_vita.c

```diff
@@ -368,20 +328,19 @@ ipmi_vita_get_vso_capabilities(struct ipmi_intf *intf)
 	}
 
 	printf("VSO Standard      : %s\n",
-		(rsp->data[3] & 0x3) == 0 ? "VITA 46.11" : "RESERVED");
+	       (rsp->data[3] & 0x3) == 0 ? "VITA 46.11" : "RESERVED");
 
 	printf("VSO Spec Revision : %d.%d\n", rsp->data[4] & 0xf,
-		rsp->data[4] >> 4);
+	       rsp->data[4] >> 4);
```

Comment:
And these `0xf` and `4`... That's a separate PR though, I guess.

Reply:
separate PR.